docs/content/operations/debugging_gloo/_index.md

```diff
@@ -49,6 +49,44 @@ Checking Proxies... OK
 No problems detected.
 ```
 
+### Create a Gloo Gateway debug report
+
+Use the `glooctl debug` command to collect Kubernetes, Gloo Gateway controller, and Envoy state information from your environment and to store it in a local `debug` directory. This information is useful to debug failures in your environment. 
+
+The command greps the following resources: 
+* Kubernetes cluster state
+* Logs from all pods in the given namespaces
+* YAML manifests of all Solo.io custom resources in the given namespaces
+* Gloo controller logs, metrics, xds snapshot, and krt snapshots
+* Envoy config dump, stats, clusters, and listeners
+
+To generate a debug report for the `gloo-system` namespace, run the following command. Note that depending on the size of you environment, it might take a few minutes for the information to be collected and stored in your local directory. 
+
+```sh
+glooctl debug -N gloo-system
```

Comment:
I think it should be lowercase, unless this is a different option than `--namespace`? but I am not sure, it is capitalized in a lot of places
```suggestion
glooctl debug -n gloo-system
```